package.json

```diff
@@ -48,7 +48,7 @@
     "bval": "~0.1.6",
     "bweb": "~0.1.8",
     "goosig": "~0.1.0",
-    "hs-client": "~0.0.6",
+    "hs-client": "github:wi-ski/hs-client#node-http-routes",
```

Comment:
This should never make it to master.

Reply:
I put this in here to make CI pass, but CI is still failing for some reason. :cry:

Reply:
For now you should use the HTTP methods on the client object, like `client.get`